paper.tex

```diff
@@ -516,19 +520,31 @@ \subsubsection{Conceptualization}
 The second position would be a central RSE, responsible for providing selected services and infrastructure.
-These central positions will be complemented by the existing RSEs organized in the network to form a pool of institutional RSEs associated with the department.
+These central positions will be complemented by the existing RSEs organised in the network to form a pool of institutional RSEs associated with the department.
 
 Drawing from the concept and considering the actual initial staff situation, a first task of the centrally funded structure is to define a basic service portfolio according to the modules described in \autoref{sec:vision}.
 In addition to the already mentioned networking and teaching cf. \autoref{sec:network} and \autoref{sec:teaching}, it seems natural to start with consultation cf. \autoref{sec:consultation}, as this allows to evaluate the potential necessities for other services such as development, infrastructure provisioning and maintenance.
-An extension of the initial service portfolio for a larger target audience requires the acquisition of further funding and installation of further positions, see below.
+An extension of the initial service portfolio for a larger target audience requires the acquisition of further funding and installation of further positions, see below (\autoref{sec:growth_of_dep}).
+
+\subsubsection{Internal communication of the department}
+The best RSE department can easily become useless if other departments as well as RSEs outside the central hub do not know about it. 
+Although an initial announcement will be done once the the RSE department concept has been approved by the institution, communication across the RSE hub, the RSE network and other departments should be done on a regular basis. 
+Such a communication should facilitate collaboration among RSEs while also keeping non-RSEs in the loop. 
+
+For communication among RSEs, some options include messaging channels (for quick communication) and forums or wikis integrated to developer platforms to have longer discussion while keeping track of the conversation, e.g.,  via issues. 
+Project management communication tools, e.g., Kanban boards, could complement the picture. 
+
+With regards to the broader community, i.e., departments within the corresponding institute, a portal including the portfolio of services and training materials around best RSE practices, would be ideal. 
+This in fact could be one of the initial infrastructure tasks carried out by the second RSE previously described (see \autoref{sec:creation_of_dep}). To communicate and advertise the portal within the center, the corresponding admins for internal communication (the people responsible for intranet news for instance) should be involved in this process.
```

Comment:
I think this example is too specific. The admins of the intranet (at our institution) are exactly that: they handle the network infrastructure, and they have next to no contact to the actual users of it. Here, they are the wrong people for this task.